scripts/README.md

```diff
@@ -2,14 +2,16 @@
 
 ## Release
 
+> Tip: You can copy raw markdown checklist below to the release Pull Request and follow it step by step marking completed items.
+
 A typical release goes like this:
 
 ### Prepare the release of the packages
 
 The following steps must be proposed as a pull request.
 
-1. Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
-   To do so, use `yarn release:changelog` The options are the following:
+1. - [ ] Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
```

Comment:
I never saw a task list that is also numbered. GitHub doesn't support out of box so " - " needs to be added. IMO I would keep only the checkboxes. I don't recall we using the numbers to refer to each item, they are there only to give a notion of a "step".

Reply:
> GitHub doesn't support out of box so " - " needs to be added

It does, " - " is needed anyway - see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#task-lists

> IMO I would keep only the checkboxes.

I just didn't want to change the formatting too much 😀 I agree that ditching numbered list would be cleaner, will update the PR